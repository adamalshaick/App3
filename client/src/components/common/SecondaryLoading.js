import React from "react";
import { MutedDots } from "./styles/StyledLoading";

const SecondaryLoading = () => {
  return (
    <div className="entry">
      <MutedDots>...</MutedDots>
    </div>
  );
};

export default SecondaryLoading;
