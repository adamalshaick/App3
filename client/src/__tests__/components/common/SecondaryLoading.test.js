import React from "react";
import SecondaryLoading from "../../../components/common/SecondaryLoading";
import renderer from "react-test-renderer";

const wrapper = renderer.create(<SecondaryLoading />).toJSON();

describe("render component", () => {
  it("renders component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
