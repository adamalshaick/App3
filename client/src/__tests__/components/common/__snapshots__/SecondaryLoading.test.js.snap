// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component renders component 1`] = `
<div
  className="entry"
>
  <div
    className="sc-htpNat kKAUuf"
  >
    ...
  </div>
</div>
`;
